index.html

```diff
@@ -1,14 +1,161 @@
+<link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;0,900;1,300;1,900&display=swap" rel="stylesheet">
+<link rel="stylesheet" href="style.css">
+  <title>Free Thinking Conference: The value of reason, logic, and evidence.</title>
 </head>
-<body>
-  <h1>hello world!</h1>
+
+<body class="body">
+  <div class="mobile-menu hide">
+    <img class="mobile-menu-close" src="images/icons/pop-close.svg" alt="close">
+    <ul class="mobile-menu-elements">
+      <li class="mobile-menu-elements-element">Program</li>
+      
+      <li class="mobile-menu-elements-element" id="about-link"><a href="#about">About</a></li>
+      <li class="mobile-menu-elements-element">Join</li>
+      <li class="mobile-menu-elements-element">Sponsor</li>
+      <li class="mobile-menu-elements-element">News</li>
+      <li class="mobile-menu-elements-element">CC Campaign</li>
+    </ul>
+  </div>
+  <div class="mainest">
+    <div class="home" id="home">
+      <div class="nav-bar-and-hero">
+        <nav class="nav-bar">
+          <div class="ham">
+            <div class="line"></div>
+            <div class="line"></div>
+            <div class="line"></div>
+          </div>
+          <a class="ham-desk" href="#home">
+            <img src="images/photos/cc_image.png" alt="Free thinking image and logo" class="cc_image">
+          </a>
+          <ul class="nav-desk-right">
+            <li class="nav-desk"><a href="#about">About</a></li>
+            <li class="nav-desk">Program</li>
+            <li class="nav-desk">Join</li>
+            <li class="nav-desk">Sponsor</li>
+            <li class="nav-desk">News</li>
+            <li class="nav-desk">CC Campaign</li></ul>
+        </nav>
+
+        <section class="hero">
+          <div class="hello">"Hello, Free Thinkers!"</div>
+          <h1 class="title">FREE THINKING SUMMIT GLOBAL 2020</h1>
+          <p class="description">Welcome to the third global summit of Free Thinking and Exceptisism when we will share the value of reason, logic and evidence. It will take place in Korea this Oct. 2020.</p>
+          <div class="date">2020.10.15(THU) ~ 16(FRI)</div>
+          <footer class="national">@ National Museum of Korea, Art Center Nabi and more</footer>
+        </section>
+      </div>
+      <section class="program">
+        <header>
+          <div class="text">Main Program</div>
+          <div class="horizontal-rule"></div>
+        </header>
+        <ul class="elements">
+          <li class="element">
+            <img class="icon" src="images/icons/program_lecture.svg" ></img>
+            <div class="title">Lecture</div>
+            <h5 class="description">Lorem ipsum dolor sit amet consectetur adipisicing elit. Impedit id rerum ex neque non.</h5>
+          </li>
+          <li class="element">
+            <img class="icon" src="images/icons/program_cc_exhibition.svg"></img>
+            <div class="title">CC Exhibition</div>
```

Comment:
- [ ] Dear @danifromecuador, Please consider adding an `alt` text for all your images.  This has a lot of advantages when it comes to semantic HTML. Screen readers and SEO need such information to pars your webpage and understand its content.